src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -592,6 +612,13 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         const int newDpi = static_cast<int>(static_cast<double>(USER_DEFAULT_SCREEN_DPI) *
                                             _compositionScale);
 
+        if (_renderEngine)
+        {
+            // Make sure to call these before we call TriggerFontChange
+            _renderEngine->SetFontFeatures(_fontFeatures);
+            _renderEngine->SetFontAxes(_fontAxes);
+        }

```

Comment:
You should be able to call this directly from `UpdateSettings`. Then you won't have to hold onto these two new members.

Reply:
Wouldn't `UpdateSettings` just call the two new members as well? I'm not sure how moving the calls to `UpdateSettings` removes the need for them

Reply:
Oh I thought `UpdateSettings` would be called anyways...